hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/TestableFederationInterceptorREST.java

```diff
@@ -51,4 +70,48 @@ protected void registerBadSubCluster(SubClusterId badSC) {
     interceptor.setRunning(false);
   }
 
+  protected void setupResourceManager() throws IOException {
+    try {
+      if (mockRM != null) {
```

Comment:
You can move some of this before the try

Reply:
I will fix it.